Dockerfile

```diff
@@ -2,20 +2,10 @@ FROM node:7.0.0
 
 COPY . deck/
 
-RUN useradd -ms /bin/bash node
-
-RUN chown -R node deck
-
-RUN chown -R node /usr/
-
-ENV HOME /home/node
-
-USER node
-
 WORKDIR deck
 
-RUN rm -rf .git
+RUN npm install --production
 
-RUN npm install
+RUN mv ./node_modules ./node_modules.tmp && mv ./node_modules.tmp ./node_modules && npm install
```

Comment:
My favorite thing about this PR is I was just trying to get the Docker build to work with Node 7 and the rest of this fell out from that because the bower install barfs on Docker for some reason.